src/routes/backingImage/BackupBackingImage.js

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect, useState, useRef } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'dva'
+import { Row, Col } from 'antd'
+import cx from 'classnames'
+import { Filter } from '../../components/index'
+import BackupBackingImageList from './BackupBackingImageList'
+import BackupBackingImageBulkActions from './BackupBackingImageBulkActions'
+import RestoreBackupBackingImageModal from './RestoreBackupBackingImageModal'
+import { filterBackupBackingImage } from './utils'
+import styles from './BackupBackingImage.less'
+
+const BackupBackingImage = ({ backingImage, loading, dispatch, location, className }) => {
+  const containerRef = useRef(null)
+  const actionBarRef = useRef(null)
+  const [tableBodyHeight, setTableBodyHeight] = useState(0)
+
+  useEffect(() => {
+    const calculateTableBodyHeight = () => {
+      if (containerRef.current) {
+        const containerHeight = containerRef.current.getBoundingClientRect().height || 0
+        const actionBarHeight = actionBarRef.current.getBoundingClientRect().height || 0
+        const tableHeaderHeight = document.querySelector('.ant-table-thead')?.getBoundingClientRect().height || 0
+        const paginationHeight = 48
+        const containerPadding = 24
+
+        const tableBodyHeightValue = containerHeight - actionBarHeight - tableHeaderHeight - paginationHeight - containerPadding
+        setTableBodyHeight(tableBodyHeightValue)
+      }
+    }
+
+    calculateTableBodyHeight()
+    window.addEventListener('resize', calculateTableBodyHeight)
+
+    return () => {
+      window.removeEventListener('resize', calculateTableBodyHeight)
+    }
+  }, [])
```

Comment:
Only the 1useEffect1 for calculating the table body height was added, the rest of the logic is moved from `backingImage/index.js`.